fast_llm/data/data/gpt/data.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         vocab_size: int,
         max_sequence_length: int,
         cross_document_attention: bool = True,
+        prediction_heads: int = 1,
```

Comment:
The dataset shouldn't have to worry about how it's used by the model, I suggest to rename to something more related to the present class. Maybe `extra_tokens`?